contents/docs/index.md

```diff
@@ -3,28 +3,30 @@ title: PostHog Documentation
 sidebar: Docs
 showTitle: true
 ---
-
-These docs explain how to use PostHog.
+<br>
+These Docs explain how to deploy, use, and contribute to PostHog.
 
 ## Philosophy
 
-We believe the era of 3rd party product analytics software will eventually come to end.
-
-We strongly believe 3rd party analytics don't work anymore in a world of Cookie laws, GDPR, CCPA and lots of other 4 letter acronyms. There should be an alternative to sending all of your customer's personal information and usage data to 3rd parties.
+We believe the era of third-party product analytics software will eventually come to end.
 
-PostHog gives you full control over all your customer's data, while being able to do powerful analytics.
+In our view, third-party analytics does not work anymore in a world of Cookie Laws, GDPR, CCPA, and many other four-letter acronyms. There should be an alternative to sending all of your customer's personal information and usage data to third-parties like Google.
 
-We see lots of larger companies who can't send their data to 3rd party tracking services for cost or privacy reasons. They end up with a complex data pipeline into a data lake and then an analytics platform on top. Data analysts end up being the only people who understand it.
+PostHog gives you full control over all the data from your users, while being able to do powerful analytics.
```

Comment:
+ without having to set up a data pipeline, data lake, visualization tools and hiring data analysts!